src/main/java/uk/ac/ebi/ena/readtools/loader/fastq/DataSpot.java

```diff
@@ -70,28 +70,28 @@
     static final long serialVersionUID = 1L;
     static final protected char[] line_separator = System.getProperty ( "line.separator" ).toCharArray();
     
-    @FeedableData( method = "feedBaseName" )
+    @ProducibleData( method = "feedBaseName" )
     public String bname; // name for bases 
         
-    @FeedableData( method = "feedBases" )
+    @ProducibleData( method = "feedBases" )
     public String bases;// bases
         
-    @FeedableData( method = "feedQualName" )
+    @ProducibleData( method = "feedQualName" )
     public String qname; // name for qualities // no + here, it will be in stop symbols
 
-    @FeedableData( method = "feedQuals" )
+    @ProducibleData( method = "feedQuals" )
     public String quals;
 
     
-    @FeedableDataChecker
+    @ProducibleDataChecker
     public void
-    checkFeed() throws QualityNormaizationException, DataFeederException
+    checkFeed() throws QualityNormaizationException, DataProducerException
     {
         if( !params.allow_empty )
         {
             if( null == bases || null == quals 
                 || 0 == bases.length() || 0 == quals.length() )
-                throw new DataFeederException( params.line_no, "Empty lines not allowed" );
+                throw new DataProducerException( params.line_no, "Empty lines not allowed" );
         }
         normailzer.normalize( quals );
     }

```

Comment:
Check if checkfeed is used after removing CompleteGenomics. If not used then remove.